release/README.md

```diff
@@ -84,11 +84,20 @@ EOF
 
     1. The YAMLs are now released! Anyone installing Tekton Chains will now get the new version. Time to create a new GitHub release announcement:
 
+    1. Find the Rekor UUID for the release
+    ```bash
+    RELEASE_FILE=https://storage.googleapis.com/tekton-releases/chains/previous/${VERSION_TAG}/release.yaml
+    CONTROLLER_IMAGE_SHA=$(curl $RELEASE_FILE | egrep 'gcr.io.*controller' | cut -d'@' -f2)
+    REKOR_UUID=$(rekor-cli search --sha $CONTROLLER_IMAGE_SHA | grep -v Found | head -1)
+    echo -e "CONTROLLER_IMAGE_SHA: ${CONTROLLER_IMAGE_SHA}\nREKOR_UUID: ${REKOR_UUID}"
+    ```
+
     1. Create additional environment variables
 
     ```bash
     CHAINS_OLD_VERSION=# Example: v0.11.1
-    TEKTON_PACKAGE=tektoncd/chains
+    CHAINS_RELEASE_NAME=$CHAINS_OLD_VERSION
```

Comment:
Just sanity checking - should this actually be the old version, or should this be the new version we are trying to release (i.e. `v0.11.2`)?

Reply:
good catch! fixed